cdk-codepipeline-cross-account-deployment/README.md

```diff
@@ -0,0 +1,100 @@
+# CDK CodePipline Cross-Account Deployment
+
+This pattern deploys a CodePipline using CodeCommit and CodeBuild. From this pipeline it deploys a CloudFront distribution in front of an S3 Bucket to serve as a static website, across 2 accounts. This is an example of how you can use one pipline to deploy the same stack across multiple environments.
+
+Note that CloudFront and S3 are used in this pattern for demonstration purposes, you can use this pattern with any stack you wish to deploy.
+
+![Pattern architecture](img/serverless-pattern.png)
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the AWS Pricing page for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create AWS account(s)](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Cloud Development Kit](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html) (AWS CDK >= 2.2.0) Installed
+
+## Language
+
+Python
+
+## Framework
+
+CDK
+
+## How It Works
+
+After cloning this pattern, you will bootstrap 2 AWS environments with CDK. An environment is an Account-Region pair. This example uses 2 different accounts as a way to demonstrate deploying the same CDK stack(s) across accounts i.e. Development and Production. The pipeline will only deploy in the account you designate in the app.py file.
```

Comment:
I do agree that providing additional configuration, such as context in the cdk.json file to differentiate between the environments would normally be useful and cleaner, but in this case, we are only deploying 1 time from local, into the primary (environment with pipeline). After this, all deployments can be done through the pipeline through the CI/CD process. Therefore, the additional configuration wouldn't be needed, especially since we are only deploying to the one account locally the one time. Once the pipeline is initialized, the example static site will be deployed to the secondary account from the pipeline.

I added more info about this in the next commit.